### DISCUSSION TITLE
why don't you use vitest?

### DISCUSSION THREAD

**Discussion by jason-liu22** [mirumee/saleor-dashboard#2827] [Q&A] [answered]
This project uses Vite, and Jest for unit test.
Why don't you use [Vitest](https://vitest.dev/)?
What's benefit from Jest over Vitest?

**Comment by kzuraw** [comment, marked as answer]
Hi @endo-aki22 and thank you for your question 👋🏻

We have an issue created for migrating from Jest (old library) to Vitest: https://github.com/saleor/saleor-dashboard/issues/2740. I think we need to refine and prioritize this but definitely, this is something we are looking into.
